README.md

```diff
@@ -13,3 +13,31 @@ Contents of this repo:
 ## License
 
 (c) Google, 2015. Licensed under [Apache-2](LICENSE) license.
+
+## Usage
+
+First, create an unencrypted private key for use with account registration. This can be done using OpenSSL:
```

Comment:
this step as completely optional as goacme can generate an account key for you and store it on disk.